course/management/commands/delete_instance.py

```diff
@@ -0,0 +1,53 @@
+from django.core.management.base import BaseCommand
+from course.models import CourseInstance
+import datetime
+
+class Command(BaseCommand):
+    help = "Delete course instance along with related data (exercises, submissions, etc)."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-c',
+            '--course-instance',
+            type=int,
+            help='Course instance ID to be deleted.',
```

Comment:
With the argument `nargs='*'`, multiple course instance ids could be given in the command line at once. Then, the `handle` method would read a list of ids.
https://docs.python.org/3/library/argparse.html#nargs

Reply:
right. now multiple IDs are supported.